cupyx/tools/install_library.py

```diff
@@ -174,6 +182,9 @@ def _make_nccl_record(
     }
 
 
+_nccl_records.append(_make_nccl_record(
```

Comment:
Could you use 2.10.3?

Reply:
Let's do this in separate NCCL 2.10 PR 😄

Reply:
Sure!